src/pages/register-a-beacon/more-details.tsx

```diff
@@ -39,15 +41,22 @@ const definePageForm = ({ moreDetails }: FormSubmission): FormManager => {
 const MoreDetails: FunctionComponent<FormPageProps> = ({
   form,
   showCookieBanner,
+  flattenedRegistration,
 }: FormPageProps): JSX.Element => {
+  const environment = flattenedRegistration.environment;
+  const previousPageUrlMap = {
+    [Environment.MARITIME]: PageURLs.vesselCommunications,
+    [Environment.AVIATION]: PageURLs.aircraftCommunications,
+    [Environment.LAND]: PageURLs.landOtherCommunications,
+    [Environment.OTHER]: PageURLs.landOtherCommunications,
+  };
```

Comment:
Back button functionality to be replaced by conditional component: if client-side JS available, send user to browser back, else send to previous page in form flow (not check-your-details).  (This change out-of-scope for this PR)

Reply:
@zackads Raised an issue on the project backlog to cover this off: https://github.com/mcagov/beacons-webapp/issues/207